runtime/testing/e2e/a11y.js

```diff
@@ -0,0 +1,2 @@
+const analyzer = require('../../../lib/a11y-analyzer');
+module.exports = { SkyA11y: analyzer };
```

Comment:
This is essentially an adaptor to meet the NodeJS requirements to stub out `protractor` and `axe-axe-webdriverjs` in the unit tests, and our consuming SPAs requirement to import a TypeScript class.